packages/workbox-routing/src/lib/router.js

```diff
@@ -41,6 +44,7 @@ import normalizeHandler from './normalize-handler';
  * // Both the routes are registered with the router, and any requests that
  * // don't match either route will be handled using the default NetworkFirst
  * // strategy.
+ * // If a request matches both routes, the "images" handler will be used. 
```

Comment:
Could you tweak the wording slightly here to 

"If a request matches both routes, the last route will be used to handle the request, in this case, the "images" handler would take precedence."

Reply:
Sure